File: CarRental.Entities/DTOs/RentalDetailDto.cs
using CarRental.Core.Entities;
using CarRental.Entities.Concrete;
using System;
using System.Collections.Generic;
using System.Text;

namespace CarRental.Entities.DTOs
{
    public class RentalDetailDto : IDto
    {
        public int Id { get; set; }
        public int CustomerId { get; set; }
        public string CustomerName { get; set; }

        public int CarId { get; set; }
        public string Brand { get; set; }
        public string Model { get; set; }
        public string Color { get; set; }
        public string CarType { get; set; }
        

        public DateTime RentDate { get; set; }
        public DateTime? ReturnDate { get; set; }
        public double? TotalPrice { get; set; }
        public float? Discount { get; set; }
    }
}

File: carRental/src/app/models/rentalDetailDto.ts
export interface RentalDetailDto{
    id:number;
    customerName:string;
    carId:number;
    brand:string;
    model:string;
    color:string;
    carType:string;
    rentDate:Date;
    returnDate:Date;
    discount:number;
    totalPrice:number;
}